### PR TITLE
Update sqlalchemy

### PR DESCRIPTION
**Pull request by loicgasser**
Based on https://github.com/geoadmin/mf-chsdi3/pull/1143 (To be merged first)

This PR updates psycopg2, SQLAlchemy, GeoAlchemy and Papyrus to their latest versions.

With the update of postgresql and postigis (2+), I thought updating those librairies made sense.

We moved forward to geolachemy2 and I am using a work Eric has recently made to update papyrus accordingly.
https://github.com/elemoine/papyrus/commit/b630639f9b9e69f2bfeb9c18b8e5f64039d3b395
https://github.com/elemoine/papyrus/issues/23

For SQLAlchemy, here are the changes:
http://docs.sqlalchemy.org/en/rel_0_8/changelog/migration_08.html
http://docs.sqlalchemy.org/en/rel_0_9/changelog/migration_09.html

And GeoAlchemy
http://geoalchemy-2.readthedocs.org/en/latest/migrate.html#migrate

I tested everything and it seems pretty solid.

### CODE REVIEW

**Comment by loicgasser** [comment]
Rebased + adapted the new models

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -1301,7 +1384,8 @@ class geometaNfgeom(Base, Vector):
     telefon = Column('telefon', Text)
     email = Column('email', Text)
     bgdi_created = Column('bgdi_created', Text)
-    the_geom = GeometryColumn('the_geom_gen50', Geometry(dimension=2, srid=21781))
+    the_geom = Column(Geometry(geometry_type='GEOMETRY',
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: Oups I need to change that.

**Comment by loicgasser** [comment]
Ok @gjn we're good to go.

**Comment by gjn** [comment]
Pep errors.
```
chsdi/models/vector/stopo.py:1299:32: E128 continuation line under-indented for visual indent
chsdi/models/vector/stopo.py:1323:32: E128 continuation line under-indented for visual indent
chsdi/models/vector/stopo.py:1346:32: E128 continuation line under-indented for visual indent
chsdi/models/vector/stopo.py:1369:32: E128 continuation line under-indented for visual indent
chsdi/models/vector/stopo.py:1388:32: E128 continuation line under-indented for visual indent
```

**Comment by loicgasser** [comment]
mmm are you sure you have the laste version? No error here

**Comment by loicgasser** [comment]
Everything up-to-date if I try to push force

**Comment by loicgasser** [comment]
I can't commit if pep is not happy now. ;)

**Comment by gjn** [comment]
:) that's good.

**Comment by gjn** [comment]
I deleted by local branch, fetched from github, checked out gal_update_sqlalchemy and the pep error is there when I build.

Could you verify?

**Comment by loicgasser** [comment]
sorry my bad it wasn't commited ok now

**Comment by gjn** [comment]
Thanks alot!

**Comment by gjn** [comment]
I'm installing to main now

**Pull request merged**